VAGOV-910: Fixed sidebar templates and queries

acabouet opened the pull request:
## Description
This is a fixed PR for pulling sidebar navigation content from Drupal.

## Testing done


## Screenshots


## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

acabouet committed:
VAGOV-910: Fixed sidebar templates and queries

acabouet commented:
@jbalboni see the build failing on this but I don't have jenkins access to see why. I think it's probably due to an error I'm getting with the build locally, that seems to be related to one of the liquid templates based on the trace. Error -  TypeError: Cannot read property 'offset' of undefined

jbalboni commented:
@acabouet Looks like linter errors:

![screen shot 2019-02-01 at 9 35 06 am](https://user-images.githubusercontent.com/634932/52129114-b6cb1d00-2604-11e9-992e-d64e95a67fec.png)

If you have Prettier set up in your editor, running it on those files will fix the lint errors. You can also run `yarn lint:js:fix` to fix them.

We have a pre-commit hook that runs our linter on files that have been changed, which you can install with `yarn prebuild` (I'm guessing you haven't ever needed to run `yarn build`, where it would get installed automatically.

jbalboni commented on src/site/stages/build/drupal/metalsmith-drupal.js:
```diff
@@ -74,6 +77,21 @@ function pipeDrupalPagesIntoMetalsmith(contentData, files) {
+    }
+  }
+
+  files['drupal/sidebar_navigation/index.html'] = {
```
This looks like debugging info for the Health Care sidebar items. Is that correct?

acabouet replied:
Yup correct, to print them to the console the same way y'all did for items on pages.

jbalboni reviewed (approved):
This looks good to me, just need to fix the build errors.

zurbergram committed:
VetTec: Unable to "Continue your application" for 0994 from https://staging.va.gov/education/how-to-apply/#16654 (#9494)

VetTec: Unable to "Continue your application" for 0994 from https://staging.va.gov/education/how-to-apply/

robfelder committed:
BAH VET TEC Content updates (#9500)

* Content updates

* Removing bank info functional updates to reduce size

* Updating unit tests

aurora-a-k-a-lightning committed:
Drupal: Collapsible-panel with collapsible-panel-items - templates and queries (#9503)

acabouet committed:
VAGOV-910: Fixing linting errors

jbalboni committed:
Fix temp React landing pages prod check (#9507)

U-DON committed:
Map props in SaveInProgressIntro instead of IntroductionPage. (#9488)

* Map props in SaveInProgressIntro instead of IntroductionPage.

* Removed withRouter applied from separate PR.

* Fixed VIC v2 IntroductionPage spec.

* Restored prop types.

* Removed unnecessary export.

acabouet committed:
VAGOV-910: Fix linting errors

acabouet committed:
VAGOV-910: Fixing linting errors
